.circleci/config.yml

```diff
@@ -298,6 +298,9 @@ jobs:
                 wget http://dl.fbaipublicfiles.com/habitat/objects_v0.1.zip
                 unzip objects_v0.1.zip -d data/objects/
                 rm objects_v0.1.zip
+                wget http://dl.fbaipublicfiles.com/habitat/locobot_merged.zip
+                unzip locobot_merged.zip -d data/objects/
+                rm locobot_merged.zip
```

Comment:
I understand the workflow now after reading this. I had initially unzipped locobot_merged into the data/locobot_merged folder and was surprised when the example script crashed. (I do now see that in your instructions, you do ask them to be unzipped into data/objects/)

Instead of dumping everything into a single objects folder, would it be better to unzip locobot_merged.zip into data/locobot_merged so that way each asset-dataset is contained in it's own folder?

Reply:
Hm, in this case it may be best to add another directory like we have for scenes (`data/scene_datasets/`) and do `data/object_datasets/objects/` and `data/object_datasets/locobot_merged/`.

Reply:
Changing this would break some existing user workflows I think. Let's talk about data organization and make a separate PR to update things to clarify the breaking nature of this change.

Reply:
Sounds good.